src/components/Button.js

```diff
@@ -41,15 +43,13 @@ module.exports = React.createClass({
 		};
 	},
 	render() {
+		var classes = this.props.sheet.classes;
```

Comment:
In fact react-jss is not always required, it depends on the size of an application. Because basically react-jss is an optimization. Most applications will be fine by just having all the styles attached all the time as it is the case when using normal css.